### PR TITLE
fix(markers): fix problem with old groups

### PR DESCRIPTION
**Pull request by wpanas**
Fixed problem from the issue #206 with old groups which were referencing to destroyed DOM elements.

@nmccready I don't know if any changes in tests are needed. If you can give me instructions, I will be glad to implement them.

### CODE REVIEW

**Comment by nmccready** [comment]
So master build is failing. I will need to fix this first.

**Comment by nmccready** [comment]
Please merge with master build should pass now.

**Comment by nmccready** [comment]
@wpanas see above

File: src/directives/markers.js [hunk 1]
```diff
@@ -233,6 +234,7 @@ angular.module('ui-leaflet').directive('markers',
 
                 getLayers().then(function(layers) {
                     var _clean = function(models, oldModels){
+                        resetMarkerGroups();
```
Comment (nmccready) [inline src/directives/markers.js]: This does resolve the problem, but does it cause all the markers to flash / rebuild?

Comment (wpanas) [inline src/directives/markers.js]: As far as I know, it doesn't cause markers to flash.

**Comment by nmccready** [comment]
LGTM

**Pull request merged**